test/GADM.jl

```diff
@@ -109,3 +109,52 @@ end
         @test diff < 0.01
     end
 end
+
+function getmeshespolygon(geometry)
+    #converts [Float, Float] to Meshes Point object
+    topoint2f = x -> Meshes.Point(x)
+    coordinates = GeoInterface.coordinates(geometry)
+
+    if string(ArchGDAL.getgeomtype(geometry)) === "wkbMultiPolygon"
+        outer = map(topoint2f, first(coordinates[1]))
```

Comment:
Since the design always returns a multipolygon, we just need to loop over the array and construct a bunch of polygons:

```
polygons = [Polygon(first(cs), cs[2:end]) for cs in coordinates]
```

This should simplify the rest of the code.